docker/ubuntu/Dockerfile

```
@@ -82,6 +83,7 @@ RUN  if [ -z "$WITH_TESTS" ] || [ "$WITH_TESTS" = "false" ]; then \
        WITH_TESTS_ARG=""; \
      fi \
   && conan profile detect \
+  && cd cppagent \
   && conan create . \
```

I believe we could eliminate changing directory by replacing `.` with an absolute path.

```suggestion
  && cd cppagent \
  && conan create /root/agent/cppagent \
```

---

Again, I added wrong suggestion, here’s the corrected one:

```suggestion
  && conan create /root/agent/cppagent \
```